source/deploy/system-deploy-interface.rst

```diff
@@ -0,0 +1,148 @@
+Interface between the deploy system and projects
+================================================
+
+This interface isn't always as clear-cut as it could be. For now, we'll
+try to document it as-is.
+
+Definitions
+~~~~~~~~~~~
+
+`Project`: One or more sites that share a code base. In other words,
+if sites' Django code comes from the same source control repository,
+we'll call them part of the same project.  The Twelve-Factor App `calls
+this the app <http://12factor.net/codebase>`_.
```

Comment:
And I don't know how much of our current deploy system was designed with Twelve-Factor in mind, but we at least seem to be solving some of the same problems the same way, so I thought it was interesting to mention it.